packages/frontend/src/services/StakingFarmContracts.js

```diff
@@ -1,12 +1,26 @@
+import { Account, Connection } from 'near-api-js';
+
 import {
-    wallet
-} from '../utils/wallet';
+    NETWORK_ID,
+    NODE_URL
+} from '../config';
+import { FARMING_VALIDATOR_VERSION, getValidationVersion, MAINNET, TESTNET } from '../utils/constants';
 
 // Staking Farm Contract
 // https://github.com/referencedev/staking-farm/
 export default class StakingFarmContracts {
     // View functions are not signed, so do not require a real account!
-    static viewFunctionAccount = wallet.getAccountBasic('dontcare');
+    static viewFunctionAccount = new Account(
+        Connection.fromConfig({
+            networkId: NETWORK_ID,
+            provider: {
+                type: 'JsonRpcProvider',
+                args: { url: NODE_URL + '/' },
+            },
+            signer: {}
```

Comment:
Why did this change? to remove the `signer`?

Reply:
It was to address some require cycles that were causing some imports to be `undefined`